lightningnetwork/lnd
# Lightning+pendingchannels

## Change Description
The definition of  a pending channel according to the `pendingchannels` API includes pending_open_channels. However, the total_limbo_balance description does not reflect that pending_open_channels are not included in this calculation. This updates the description to clarify that the total_limbo_balance does not include pending_open_channels.

## Steps to Test
Open a channel. While it is waiting to confirm, call `pendingchannels`. total_limbo_balance will return 0.

## Pull Request Checklist
### Testing

### Code Style and Documentation

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.
